homeassistant/components/tessie/device_tracker.py

```diff
@@ -0,0 +1,86 @@
+"""Device Tracker platform for Tessie integration."""
+from __future__ import annotations
+
+from homeassistant.components.device_tracker import SourceType
+from homeassistant.components.device_tracker.config_entry import TrackerEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.typing import StateType
+
+from .const import DOMAIN
+from .coordinator import TessieDataUpdateCoordinator
+from .entity import TessieEntity
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
+) -> None:
+    """Set up the Tessie device tracker platform from a config entry."""
+    coordinators = hass.data[DOMAIN][entry.entry_id]
+
+    async_add_entities(
+        EntityClass(coordinator)
+        for EntityClass in (
```

Comment:
```suggestion
        klass(coordinator)
        for klass in (
```

I'd use lower case here since its a local var